docs/interfaces/http.rst

```diff
@@ -535,10 +535,46 @@ error::
       ]
     }
 
-.. NOTE::
+.. rubric:: Error handling
+
+There are two kinds of error behaviors for bulk requests:
```

Comment:
The paragraph above in "Bulk errors" should also be removed. The information is otherwise duplicated

Reply:
You mean like this?
```suggestion
```

Currently, I don't see any redundant information with that sentence. Most probably, you are referring to something different, and I am just not seeing it?

I will be happy to make any changes, but I need further advise. Thanks!

Reply:
No I mean like this:

```diff
diff --git a/docs/interfaces/http.rst b/docs/interfaces/http.rst
index 790dc53b1bb..73b464511dc 100644
--- a/docs/interfaces/http.rst
+++ b/docs/interfaces/http.rst
@@ -506,39 +506,10 @@ statements are:
 .. _http-bulk-errors:
 
 Bulk errors
 -----------
 
-If a bulk operation fails, the resulting row count will be ``-2`` and the
-resulting object may contain an ``error_message`` depending on the resulting
-error::
-
-    sh$ curl -sS -H 'Content-Type: application/json' \
-    ... -X POST '127.0.0.1:4200/_sql' -d@- <<- EOF
-    ... {
-    ...   "stmt": "INSERT into locations (name, id) values (?,?)",
-    ...   "bulk_args": [
-    ...     ["Mars", 1341],
-    ...     ["Sun", 1341]
-    ...   ]
-    ... }
-    ... EOF
-    {
-      "cols": [],
-      "duration": ...,
-      "results": [
-        {
-          "rowcount": 1
-        },
-        {
-          "rowcount": -2
-        }
-      ]
-    }
-
-.. rubric:: Error handling
-
 There are two kinds of error behaviors for bulk requests:
 
 1. **Analysis error:** Occurs if the statement is invalid, either due to syntax
    errors or semantic errors identified during the analysis phase before the
    execution starts. In this case the **whole** operation fails and you'll get
```